x-pack/plugins/lists/public/exceptions/__mocks__/api.ts

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
NOTE: Deleted as 1) we want to move to putting mocks next to corresponding files and 2) wasn't being used